platform/macos/src/MGLMapView.mm

```diff
@@ -1402,10 +1383,10 @@ - (void)handlePanGesture:(NSPanGestureRecognizer *)gestureRecognizer {
         _mbglMap->cancelTransitions();
 
         if (gestureRecognizer.state == NSGestureRecognizerStateBegan) {
-            _scaleAtBeginningOfGesture = _mbglMap->getScale();
+            _zoomAtBeginningOfGesture = _mbglMap->getZoom();
         } else if (gestureRecognizer.state == NSGestureRecognizerStateChanged) {
-            CGFloat newZoomLevel = log2f(_scaleAtBeginningOfGesture) - delta.y / 75;
-            [self scaleBy:powf(2, newZoomLevel) / _mbglMap->getScale() atPoint:startPoint animated:NO];
+            CGFloat newZoomLevel = _zoomAtBeginningOfGesture - delta.y / 75;
```

Comment:
I just tested this and it's working as expected. Subtracting from a zoom level is equivalent to the previous behavior of subtracting from the log₂ of a scale.